erdgo/cmd/testBlock/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-sdk/erdgo"
+)
+
+func main() {
```

Comment:
Do you think you can allow the user to dive into the blockchain, and do `hyperblockNonce--` (e.g. if the user presses ENTER or so)? That is, to turn this example into a tiny blockchain traversal utility.

Reply:
Done